release-notes/v1_60.md

```diff
@@ -345,6 +345,22 @@ We also have experimental support for debugging Jupyter notebooks using the full
 
 ![debugging jupyter notebook](images/1_60/jupyter-debug.gif)
 
+### Python
+
+**Revamped testing interface**
+
+Thanks to the [new testing API](https://code.visualstudio.com/api/working-with-extensions/testing-extension), the Python extension has revamped its testing features.  You can now enjoy a more stable experience for test discovery, navigation and status rendering, as well as new features such as search tests, run a selection of tests, rerun last executed test, and more!
```

Comment:
I think you want to link to https://code.visualstudio.com/api/extension-guides/testing